src/Term/AliasGroup.php

```diff
@@ -19,7 +19,7 @@
 class AliasGroup implements Comparable, Countable {
 
 	/**
-	 * @var string
+	 * @var string Requested and actual language of the aliases.
```

Comment:
"requested"? That seems to be about a usecase you have for the object, not something intrinsically part of the object.